toolkits/springernature/packages/springernature-user-details/README.md

```diff
@@ -52,5 +52,6 @@ Import the enhanced settings and styles into your main stylesheet
 // enhanced.scss
 @import '@springernature/brand-context/default/scss/enhanced';
 @import '@springernature/user-details/scss/10-settings/colours';
+@import '@springernature/user-details/scss/10-settings/layout';
```

Comment:
I think this file is now out of date. @anibe posted a PR this morning that updates this file and bumps the version - https://github.com/springernature/frontend-toolkits/pull/397/files

Reply:
@jon-stevens I updated the links this morning. I think you just need to add the `springer-nature-` to the file name in the docs

Reply:
Ughh it's my fault, I thought I had merged.. If you rebase with master you should get the latest. Apologies